package.json

```diff
@@ -16,7 +16,7 @@
     "karma-phantomjs-launcher": "^1.0.2",
     "karma-safari-launcher": "~1.0.0",
     "leafdoc": "^1.4.0",
-    "leaflet": "^1.0.1",
+    "leaflet": "1.0.1",
```

Comment:
Meaningless

Reply:
I'm afraid it's a mandatory change for the build to work, since the file `build/build.js`of the leaflet repo only exists in 1.0.X versions of leaflet.

Reply:
(And that this file is required in the build process to discover all leaflet source files)